test/specs/events.spec.js

```diff
@@ -1,6 +1,13 @@
 import Chart from 'chart.js';
 
 describe('events', function() {
+	beforeEach(() => {
+		Chart.helpers.requestAnimFrameBak = Chart.helpers.requestAnimFrame;
+		Chart.helpers.requestAnimFrame = (cb) => cb();
```

Comment:
Not quite sure why this is needed?

Reply:
as far as I understood, the chart.js animations use `requestAnimFrame` to render even the first frame. Thus, it renders asynchronously. However, the tests are synchronous, thus the patch to fix it.

Reply:
Ok, that could be considered a bug in chart.js
Another option could be to disable animations.